packages/shared/providers.yaml

```diff
@@ -3700,6 +3700,31 @@ microsoft-power-bi:
         base_url: https://api.powerbi.com
     docs: https://docs.nango.dev/integrations/all/microsoft-power-bi
 
+mindbody:
+    display_name: Mindbody
+    categories:
+        - fitness
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.mindbodyonline.com/public/v6
```

Comment:
You can also omit the path in here, as base url is different for the consumer API, which according to the docs is; `https://api.mindbodyonline.com/partnergateway/consumeractivity`

You can now have `/public/v6/site/locations` as your `proxy.verification` endpoint

Reply:
Okay i've removed the base path